### PR TITLE
added paws-x dataset

### PR DESCRIPTION
**Pull request by bhavitvyamalik**
Added paws-x dataset. Updating README and tags in the dataset card in a while

### CODE REVIEW

File: datasets/paws-x/paws-x.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+        for config_name in _DATA_OPTIONS
+    ]
+
+    DEFAULT_CONFIG_NAME = "en"  # It's not mandatory to have a default configuration. Just use one if it make sense.
+
+    def _info(self):
+        features = datasets.Features(
+            {
+                "id": datasets.Value("int32"),
+                "sentences": datasets.features.Sequence({"text": datasets.Value("string")}),
```
Comment (lhoestq) [inline datasets/paws-x/paws-x.py:99]: could you use two columns `sentence1` and `sentence2` ? 
we're already doing this for other paws-x

File: datasets/paws-x/paws-x.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+    BUILDER_CONFIGS = [
+        PAWSXConfig(
+            name=config_name,
+            description=(f"This config contains samples in {config_name}."),
+        )
+        for config_name in _DATA_OPTIONS
+    ]
+
+    DEFAULT_CONFIG_NAME = "en"  # It's not mandatory to have a default configuration. Just use one if it make sense.
```
Comment (lhoestq) [inline datasets/paws-x/paws-x.py:93]:
```suggestion
```
I'm not sure it makes sense to put one language forward compared to the others

**Review by lhoestq — COMMENTED** [review]
Nice ! Thank you for adding this dataset :) 

I left a few comments

Also could you make the dummy_data.zip files a bit lighter ? 
To do so you can for example take a look inside the dummy_data.zip file of `en` and remove all the files that don't belong to the `en` configuration

File: datasets/paws-x/paws-x.py (hunk 1 above)
Comment (bhavitvyamalik) [reply datasets/paws-x/paws-x.py:99]: Something like this you mean?
```
features = datasets.Features(
            {
                "id": datasets.Value("int32"),
                "sentence1": datasets.Value("string"),
                "sentence2": datasets.Value("string"),
                "label": datasets.features.ClassLabel(names=["0", "1"]),
            }
        )
```

File: datasets/paws-x/paws-x.py (hunk 2 above)
Comment (bhavitvyamalik) [reply datasets/paws-x/paws-x.py:93]: Sure, I'll make that change. Let the user decide what they want

File: datasets/paws-x/README.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+languages:
+- de
+- en
+- es
+- fr
+- ja
+- ko
+- zh
+licenses:
+- other-The dataset may be freely used for any purpose, although acknowledgement of
```
Comment (yjernite) [inline datasets/paws-x/README.md:17]: This should only be `other`, the free text description needs to be in the paragraph at the end of the file

Comment (bhavitvyamalik) [reply datasets/paws-x/README.md]: Hi @yjernite,
I made that change in README and when I pushed the code few tests failed. Can you please look into it? Thanks!

**Review by lhoestq — APPROVED** [review]
Thanks ! LGTM :)

**Pull request merged**